src/bijectors/coupling.jl

```diff
@@ -118,6 +118,20 @@ Combines `x_1`, `x_2`, and `x_3` into a single vector.
 """
 @inline combine(m::PartitionMask, x_1, x_2, x_3) = m.A_1 * x_1 .+ m.A_2 * x_2 .+ m.A_3 * x_3
 
+function ChainRulesCore.rrule(::typeof(combine), m::PartitionMask, x_1, x_2, x_3)
+    prj = map(ChainRulesCore.ProjectTo, (x_1, x_2, x_3))
+
+    function _transform_ordered_adjoint(ΔΩ)
```

Comment:
Can we use maybe the name
```suggestion
    function combine_pullback(ΔΩ)
```
here? I think this would be a bit more in line with the usual CR conventions.

Reply:
> Can we use maybe the name

Ooops, copy/paste error :-)